app/views/layouts/application.html.erb

```diff
@@ -82,7 +82,7 @@
     </div>
     <div id="footer">
       <div class="footer_wrapper">
-        <div class="part"><a href="/<%=I18n.locale.to_s%>/pages/privacy_policy">Privacy Policy</a></div>
+        <div class="part"><a href="https://www.zikaronbasalon.com/terms">Terms and Conditions</a></div>
```

Comment:
1. `Terms and Conditions` should be translated
2. is there a hebrew and english version of terms and conditions?
also, please change the base branch to develop instead